src/macaron/__main__.py

```diff
@@ -118,6 +119,31 @@ def verify_policy(verify_policy_args: argparse.Namespace) -> int:
     return os.EX_USAGE
 
 
+def verify_repo_finder() -> int:
+    """Verify the functionality of the remote API calls used by the repo finder.
+
+    Functionality relating to Java artifacts is not verified for two reasons:
+    - It is extremely unlikely that Maven central will change its API or cease operation in the near future.
+    - Other similar repositories to Maven central (internal Artifactory, etc.) can be provided by the user instead.
+    """
+    # Verify deps.dev for a Python package
+    repo_finder = RepoFinderDepsDev("pypi")
+    urls = []
+    # Without version
+    urls.append(repo_finder.create_urls("", "packageurl-python", ""))
+    # With version
+    urls.append(repo_finder.create_urls("", "packageurl-python", "0.11.1"))
+    for url in urls:
+        logger.debug("Verifying: %s", url[0])
+        metadata = repo_finder.retrieve_metadata(url[0])
+        if not metadata:
+            return 1
```

Comment:
For this method. Instead of returning integer as exit code. Let's use the return code constant provided in the `os` library [here](https://docs.python.org/3/library/os.html#os.EX_OK).
For this method. I think for 0 could be replaced by `os.EX_OK` and 1 could be replaced by `os.EX_UNAVAILABLE` ? Please feel free to suggest another constant if it's more suitable.